repos/system_upgrade/el7toel8/actors/commonleappdracutmodules/libraries/modscan.py

```diff
@@ -47,6 +47,13 @@ def _create_dracut_modules():
 
 def _create_initram_packages():
     required_pkgs = _REQUIRED_PACKAGES
+
+    with open('/proc/cmdline') as f:
+        fips = 'fips=1' in f.read()
+
+    if fips:
+        required_pkgs.append('dracut-fips')
```

Comment:
This won't work because the upgrade initramfs we create is based on RHEL 8, and there's no dracut-fips on RHEL 8 anymore: https://github.com/ComplianceAsCode/content/pull/3623
The functionality of the `dracut-fips` package is provided by the `crypto-policies` package and the `fips-mode-setup` tool on RHEL 8.